lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -68,6 +68,15 @@ class DetailsRenderer {
       case 'table':
         // @ts-ignore - TODO(bckenny): Fix type hierarchy
         return this._renderTable(/** @type {TableDetailsJSON} */ (details));
+      case 'list':
+        return this._renderList(
+          // @ts-ignore
```

Comment:
is this the same `TODO:` reason as the above?

Reply:
Yes, added that. I think @brendankenny [has fixed these todos in this PR](https://github.com/GoogleChrome/lighthouse/pull/7192/files).